lib/LoadingOverlay/index.js

```diff
@@ -1,24 +1,40 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import cx from 'classnames';
 import loadingSVG from '../../assets/loading.svg';
 
-const LoadingOverlay = ({ fixed, hideSVG }) => {
+const LoadingOverlay = ({ fixed, hideSVG, percentageText }) => {
   const className = cx('loading-overlay', {
     'loading-overlay--fixed': fixed
   });
 
-  return <div className={className}>{!hideSVG && loadingSVG()}</div>;
+  return (
+    <div className={className}>
+      <div className="loading-overlay__content">
+        {!hideSVG && loadingSVG()}
+        {percentageText && (
+          <Fragment>
+            <p className="h-margin-bottom-quarter h-margin-top-clear">
+              Loading
+            </p>
+            <p className="h-margin-clear">{percentageText}</p>
+          </Fragment>
+        )}
+      </div>
+    </div>
+  );
 };
 
 LoadingOverlay.propTypes = {
   fixed: PropTypes.bool,
-  hideSVG: PropTypes.bool
+  hideSVG: PropTypes.bool,
+  percentageText: PropTypes.string
```

Comment:
💚💚💚 What do you think to passing `percentageLoaded` as a number rather than a string, and doing the string conversion/adding of `%` in this component?

Might save a little time having to do the conversion and postfix the % every time we use this component